posts/2017-06-02-devops-serverless-variables.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "DevOps Use Cases With Serverless Variables"
+description: Serverless Framework Engineer Eslam Hefnawy explores creative use cases for using Serverless Variables to optimize and automate operations.
+date: 2017-05-01
+layout: Post
+thumbnail:https://avatars2.githubusercontent.com/u/2312463?v=3&s=60
+authors:
+  - eahefnawy
+---
+
+
+The Serverless Framework has a powerful built-in variable system that helps secure your sensitive data and cab be used to keep even the most complex configuration simple and manageable. We pretty comprehensive coverage of all of the features of the variable system in [our documentation](https://serverless.com/framework/docs/providers/aws/guide/variables/#variables), but when combined together in creative ways these features enable some extremely powerful workflows and real world use cases. 
```

Comment:
`cab be used` --> `can be used`

`We pretty comprehensive coverage ...` --> `We have pretty comprehensive coverage...`